compose/compose-file.md

```diff
@@ -1409,6 +1410,29 @@ refer to it within the Compose file:
         external:
           name: actual-name-of-network
 
+## Extra configuration reference
```

Comment:
@shin- @dnephin below is the one aspect of this PR that never made it in to the current docs. It's the addition of an `Extra configuration reference` topic. Can you all look at this content change, and let me know if I should add this to the [current Compose file reference](https://docs.docker.com/compose/compose-file/)?

Thanks, Vicky